src/renderer/views/update/UpdateView.tsx

```diff
@@ -15,7 +15,7 @@ type UpdateViewProps = {
 };
 
 const UpdateView = ({ state, variant, progress }: UpdateViewProps) => {
-  // FIXME: file lock이 제대로 걸려있지 않아서 파일을 여러 번 받아서 프로그레스가 뒤로 가는 경우가 있습니다.
+  // FIXME: files downloaded multiple times because of improper file lock, causing progress going backward.
```

Comment:
A GFI candidate. Also text:
```suggestion
  // FIXME: Some files were downloaded multiple times because of improper file lock, causing progress to go backward.
```